app/assets/javascripts/initializers/initializeAllFollowButts.js

```diff
@@ -94,7 +94,26 @@ function handleOptimisticButtRender(butt) {
     showModal('follow-button');
   } else {
     // Handles actual following of tags/users
-    assignState(butt, butt.dataset.verb);
+    try {
+        //lets try grab the event buttons info data attribute user id
+        let evFabUserId = JSON.parse(butt.dataset.info).id;
+        //now for all follow action buttons
+        document.querySelectorAll('.follow-action-button').forEach(function(fab){
+            try{
+                //lets check they have info data attributes
+                if( fab.dataset.info ){
+                    //and attempt to parse those, to grab that buttons info user id
+                    let fabUserId = JSON.parse(fab.dataset.infos).id;
```

Comment:
`.infos` is a typo. It should be `.info` right?

Reply:
Ooops, yes sorry you're spot on - I think I left my test var in there 😅